zebra-consensus/src/block/check.rs

```diff
@@ -131,9 +134,25 @@ pub fn subsidy_is_valid(block: &Block, network: Network) -> Result<(), BlockErro
         // Funding streams are paid from Canopy activation to the second halving
```

Comment:
Can you open a ticket to delete the founders reward code?

It never runs on any blocks, because pre-Canopy blocks are checkpointed.
So we should clean it up eventually.

Reply:
https://github.com/ZcashFoundation/zebra/issues/3026